Apps/Src/Amps.c

```diff
@@ -134,14 +140,14 @@ void Amps_Calibrate(void) {
 	OS_ERR err;
 
 	// initial calibration
-	LTC2315_Calibrate();
+	//LTC2315_Calibrate();
 	Amps_UpdateMeasurements();
 
 	// keep calibrating until we read 0 Amps
 	OSTimeDly(1, OS_OPT_TIME_DLY, &err);
 	Amps_UpdateMeasurements();
 	while (Amps_GetReading() != 0) {
-		LTC2315_Calibrate();
+		//LTC2315_Calibrate();
```

Comment:
You just comment out `LTC2315_Calibrate()`` when you could surround it with `#ifndef` guards or just replace the function call with a log that says "Calibrating LTC22315". This will prevent the code from working when not in simulation.